vllm/spec_decode/spec_decode_worker.py

```diff
@@ -120,6 +120,7 @@ def create_worker(
         else:
             draft_parallel_config: ParallelConfig = draft_worker_kwargs[
                 'parallel_config']
+
```

Comment:
Seems have some noise in formatting. If possible, let's not touch `spec_decode_worker.py` and `model_runner.py` in this PR to reduce possible conflicts.

Reply:
removed